From 27f6119af76011d6cbc2e40468a804b6bef31651 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Fri, 25 May 2012 11:47:39 +0200
Subject: [PATCH] patch: wmclass

Squashed commit of the following:

commit d42b7a9dbe397a301373e3cbaa589540a1475a0b
Author: Nils Philippsen <nils@redhat.com>
Date:   Fri May 25 11:45:48 2012 +0200

    set program name -> wmclass to match desktop file name
---
 src/xsane.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/xsane.c b/src/xsane.c
index 775610e..5177758 100644
--- a/src/xsane.c
+++ b/src/xsane.c
@@ -6169,6 +6169,9 @@ int main(int argc, char **argv)
     xsane.ipc_pipefd[1] = 0;
   }
 
+  /* Set program name -> wmclass to match desktop file name */
+  g_set_prgname("xsane");
+
 #if 0
   bindtextdomain(PACKAGE, STRINGIFY(LOCALEDIR));
   textdomain(PACKAGE);         
-- 
1.7.10.2

